src/style.css

```diff
@@ -1,3 +1,116 @@
+:root {
+  --nav-color: #00acdc;
+}
+
 body {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+
+  font-size: 12px;
+  font-style: normal;
+  line-height: normal;
+  text-transform: uppercase;
+
+  background-color: #d8e0e044;
+}
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*:before,
+*:after {
+  box-sizing: inherit;
+}
+
+body,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+ol,
+ul {
   margin: 0;
+  padding: 0;
+  font-weight: normal;
+}
+
+ol,
+ul {
+  list-style: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+h1 {
+  font-size: 0;
+  width: 1px;
+  height: 1px;
+  display: inline-block;
+  overflow: hidden;
+  position: absolute !important;
+  border: 0 !important;
+  padding: 0 !important;
+  margin: 0 !important;
```

Comment:
No need to use `!important`
It's a bad practice

Reply:
It's a standard approach to hide <h1> tag and to search robots can find our page. Mihailo told us at one of the lectures.
For the education, I can delete important.

Reply:
It's a standard approach to hide <h1> tag and to search robots can find our page. Mihailo told us at one of the lectures.
For the education, I can delete important.

Reply:
It's a standard approach to hide h1 tag and to search robots can find our page. Mihailo told us at one of the lectures.
For the education, I can delete important.